CHANGELOG.md

```diff
@@ -2,33 +2,24 @@
 
 ## Current develop
 
-### Updated `hdf5` handling in the build
-
-### Fixed (Repair bugs, etc)
-
```

Comment:
I think git munged the formatting of the change log at some point. I tried to clean it up here since I noticed it.